src/log_level.rs

```diff
@@ -44,7 +44,10 @@ impl std::str::FromStr for LogLevel {
             "quiet" => Ok(Self::Quiet),
             "info" | "all" => Ok(Self::Info),
             "error" => Ok(Self::Error),
-            loglevel => Err(format!("I don't know the log level of {:?}", loglevel)),
+            loglevel => Err(format!(
+                "Unrecognized log level {:?}. Supported levels are 'quiet', 'info' and 'error'.",
```

Comment:
I wonder if there's a way to automatically extract it. But anyway, I'm merging because it is important fix! thank you!